### ISSUE TITLE
include a more serious layout for elevations

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1380]
elevations data should be stored. On what levels? 
Locus-level, of course. But what about lots? Units? Collections?

### tracking
- [x] #1407
- [x] #1412 
- [x] #1413
- [x] #1417

### see also
- [x] #1379

**Comment by lbestock** [comment]
Lots yes.
Units no, I think (will be included in locus data - though think about a survey unit?)
Parker wanted us to think about a coordinate field for the middle of a locus, in which case he was thinking X and Y as well as Z - that's what he wanted to plug into the GIS. His current elevations pane asks for ten elevations per locus, one for each corner and one for the center at start, and likewise at finish. The ideal locus XYZ point of the sort we discussed to put in the GIS would be derived not mathematically from the center of those 10 but rather by the archaeologist's instinct (if one can get a feeling from a bunch of data, one can get a feeling from a locus...)
Also coming from Parker is that a special find has coordinates (XYZ). And for that matter so would some kinds of samples, though not all.

**Comment by urapadmin** [comment]
that needs to get more specific here.
Where (layout) are how many fields of what type needed?

**Comment by lbestock** [comment]
Loci should have the full work-up as per FAIMS. So many are needed there that it gets its own layout, and I though we decided that was necessary, too? The elevations needed are numeric fields and should be split into two sides laid out in parallel to one another as columns. One side is top elevations, the other bottom (or opening and closing). Each side has NW corner. NE corner. SE corner. SW corner. Center. For the center, to follow Parker's idea, what we need is the ability to get the X and Y into the GIS. That is the only relevant spatial data for analyzing the CM associated with the locus. So for Center points we would need to add the ability to put coordinates in, and there I'm unsure what coordinate system or how to enter it so that it could easily serve that role in the GIS. Or do we need to? The point would be taken with the RTK anyway. What needs to happen is that the point be saved, not just the elevation read out to the trench supervisor, and that the GIS knows to take it as the center point for that locus when that locus's CM data are brought in. Is that a checkbox in the database "use this point as center point of the locus for GIS"? Does it need anything at all in the recording?

For lots, a single opening elevation in center is sufficient I should think. And we don't need opening and closing there - by definition all lots are within a locus and if lots are stacked vertically then the opening of one is the closing of another.

For CM, a single XYZ point is sufficient. Parker shoots those in with the RTKs and saves those points so he's got that information already in the GIS by definition. Is it useful to have it separately here? Not so clear.

**Comment by urapadmin** [comment]
I think I just really don't understand the thing with the coordinate in the middle. Spatial. Not my forte. Let's talk about it.

**Comment by urapadmin** [comment]
- [ ] some pother taksdüafd

**Comment by urapadmin** [comment]
- [ ] #1383

**Comment by urapadmin** [comment]
okay, we talked about it.
- I will implement the elevations
- we need to talk to Parker and understand more about the central coordinate, I'll leave that out for the time being

**Comment by urapadmin** [comment]
# specs
## config
visibility/show_elevations: nope/simple/full

**Comment by urapadmin** [comment]
> For lots, a single opening elevation in center is sufficient I should think. And we don't need opening and closing there - by definition all lots are within a locus and if lots are stacked vertically then the opening of one is the closing of another.

Currently, we have opening and closing elevations. You want me to drop that for a single center elevation field? Let's move that to its own ticket.

**Comment by urapadmin** [comment]
### test: recording 12.1.0
elevations are included according to the specs (see above)

**Comment by luizaogs** [comment]
See tickets #1412 and #1413. This is otherwise working as expected.

**Comment by lbestock** [comment]
There should be 10 elevations - we need center elevations, too, even if we're not doing the full coordinates for center points. A thought for layout - I like that here they look like corners rather than being a list like FAIMS. I think spatially and this is spatial information. So add the center elevations in the center, please, like "5"s on dice!

Also I can't see NW etc labeled nor a way to configure it to be labeled? I am aware, however, that I am dipping into this between meetings while you guys are working solidly on it and I am undoubtedly missing stuff.

**Comment by urapadmin** [comment]
### recording 12.2
  - #1413 
  - [ ] specs have changed: basically "elevations" is not its own tab anymore but a section within "measurements"

**Comment by mcornacchia25** [comment]
Not really sure if I need to do something here or if this is just tracking the other tickets. Everything from the other tickets worked for me.

**Comment by urapadmin** [comment]
in this one since all the tracked issues are closed and everything else is checked, we can close it.
